woocommerce/woocommerce-android
# issue/3812-variation-detail-from-order

Resolves #3812 by using deep linking to navigate to the variation detail when a variation is clicked in the order detail.

To test:

- Open order detail for an order that contains a variation
- Tap on the product variation
- Verify the variation detail appears and works correctly

Update release notes:
